inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp

```diff
@@ -87,7 +74,7 @@ MKLDNNExecNetwork::MKLDNNExecNetwork(const InferenceEngine::CNNNetwork &network,
             }
         };
 
-        if (with_cpu_x86_avx512_core() && isFloatModel) {
+        if (with_cpu_x86_avx512_core()) {
```

Comment:
Not sure that enabling bf16+int8 by default is applicable in nearest release, Needs to be discussed offline.

Reply:
Done. BF16 + INT8 mode runs only if INT8 or binary models runs then enforceBF16 key sets manually.